extensions/amp-accordion/1.0/README.md

```diff
@@ -25,7 +25,17 @@ import {defineElement as defineBentoAccordion} from '@bentoproject/accordion';
 defineBentoAccordion();
 ```
 
-### Example: Include via `<script>`
+### Include via `<script>`
+
+```html
+<script type="module" src="https://cdn.ampproject.org/bento.mjs"></script>
+<script type="nomodule" src="https://cdn.ampproject.org/bento.js"></script>
```

Comment:
```suggestion
<script type="text/javascript" nomodule src="https://cdn.ampproject.org/bento.js"></script>
```

Reply:
Is this really needed?

Reply:
("text/javascript")

Reply:
No idea where that came from! 😉 

You're right, remove it. It's not needed.